cmd/documentation/documentation.go

```diff
@@ -469,21 +348,32 @@ func GetTemplateFiles() (*template.Template, error) {
 	return tmpl, filepath.Walk(toolDir, walkFn)
 }
 
-// GetContributorList fetches a list of contributors from the github api
-// endpoint
-func GetContributorList(repo string, verbose bool) ([]Contributor, error) {
-	contents, err := common.SendHTTPRequest(context.TODO(),
-		http.MethodGet,
-		repo+GithubAPIEndpoint,
-		nil,
-		nil,
-		verbose)
-	if err != nil {
-		return nil, err
-	}
+// GetContributorList fetches a list of contributors from the Github API endpoint
+func GetContributorList(ctx context.Context, repo string, verbose bool) ([]Contributor, error) {
+	var contributors []Contributor
+	vals := url.Values{}
+	vals.Set("per_page", defaultGithubAPIPerPageLimit)
+	page := 1
+	for {
```

Comment:
```suggestion
```suggestion
	for page := 1; ; page ++ {
```

```